react-state2/react-state-2_favourite-holiday/src/App.js

```diff
@@ -1,9 +1,24 @@
-import React from "react";
+import React, { useState } from "react";
 import "./styles.css";
 
 export default function App() {
+  const [holiday, setHoliday] = useState("");
+  const [date, setDate] = useState("");
+
+  const [holidayInput, setHolidayInput] = useState("");
+  const [dateInput, setDateInput] = useState("");
+
+  function resetForm(event) {
+    setHolidayInput("");
+    setDateInput("");
+    event.target.elements.holiday.focus();
+  }
+
   function handleSubmit(event) {
     event.preventDefault();
+    setHoliday(holidayInput);
+    setDate(event.target.elements.date.value);
```

Comment:
why use a separate piece of state for the holiday instead of doing the same as you're doing for the date?